web/app.js

```diff
@@ -37,4 +37,275 @@ const NewFormButton = {
+    'application/msword',
+    'application/x-dvi',
+    'application/x-director',
+    'application/postscript',
+    'text/x-setext',
+    'application/envoy',
+    'application/fractals',
+    'x-world/x-vrml',
+    'text/plain',
+    'application/x-hdf',
+    'application/winhlp',
+    'application/mac-binhex40',
+    'application/hta',
+    'text/x-component',
+    'text/html',
+    'text/html',
+    'text/webviewhtml',
+    'application/x-iphone',
+    'application/x-internet-signup',
+    'application/x-internet-signup',
+    'application/x-javascript',
+    'application/x-latex',
+    'application/octet-stream',
+    'application/octet-stream',
+    'application/x-troff-man',
+    'application/x-troff-me',
+    'message/rfc822',
+    'message/rfc822',
+    'application/x-msmoney',
+    'application/vnd.ms-project',
+    'application/x-troff-ms',
+    'application/x-msmediaview',
+    'message/rfc822',
+    'application/oda',
+    'application/pkcs10',
+    'application/x-pkcs12',
+    'application/x-pkcs7-certificates',
+    'application/x-pkcs7-mime',
+    'application/x-pkcs7-mime',
+    'application/x-pkcs7-certreqresp',
+    'application/x-pkcs7-signature',
+    'application/pdf',
+    'application/x-pkcs12',
+    'application/ynd.ms-pkipko',
+    'application/x-perfmon',
+    'application/x-perfmon',
+    'application/x-perfmon',
+    'application/x-perfmon',
+    'application/x-perfmon',
+    'application/pics-rules',
+    'application/postscript',
+    'application/x-mspublisher',
+    'application/x-troff',
+    'application/rtf',
+    'text/richtext',
+    'application/x-msschedule',
+    'text/scriptlet',
+    'application/set-payment-initiation',
+    'application/set-registration-initiation',
+    'application/x-sh',
+    'application/x-shar',
+    'application/x-stuffit',
+    'application/x-pkcs7-certificates',
+    'application/futuresplash',
+    'application/x-wais-source',
+    'application/vnd.ms-pkicertstore',
+    'application/vnd.ms-pkistl',
+    'text/html',
+    'application/x-sv4cpio',
+    'application/x-sv4crc',
+    'application/x-troff',
+    'application/x-tar',
+    'application/x-tcl',
+    'application/x-tex',
+    'application/x-texinfo',
+    'application/x-texinfo',
+    'application/x-troff',
+    'application/x-msterminal',
+    'text/tab-separated-values',
+    'text/plain',
+    'text/iuls',
+    'text/x-vcard',
+    'x-world/x-vrml',
+    'application/vnd.ms-works',
+    'application/vnd.ms-works',
+    'application/vnd.ms-works',
+    'application/x-msmetafile',
+    'application/vnd.ms-works',
+    'application/x-mswrite',
+    'x-world/x-vrml',
+    'x-world/x-vrml',
+    'x-world/x-vrml',
+    'x-world/x-vrml',
+  ],
+
+  init() {
+    this.lastTarget = null;
+
+    this.appendOverlay();
+    this.handleDragEnter();
+    this.handleDragLeave();
+    this.handleDragOver();
+    this.handleDrop();
+  },
+
+  handleDragEnter() {
+    window.addEventListener("dragenter", (event) => {
```

Comment:
CS: here you are using quotation marks but below apostrophes. Stay with apostrophes.